src/main/java/org/folio/service/TagService.java

```diff
@@ -23,7 +23,7 @@ public class TagService {
 
   private static final String TAG_ENDPOINT = resourcesPath(TAGS);
 
-  private final RestClient restClient;
```

Comment:
this should not be deleted, in scope of your pr you should create/refactor the logic only that connected with your story, not all places in the module

Reply:
Sure, sorry, I will not do it again